src/components/AssetSelect/AssetSelectContent.tsx

```diff
@@ -53,6 +53,15 @@ const AssetSelectContent: FC<Props> = ({
 
       return 0;
     })
+    .filter((asset) => {
```

Comment:
sidenote - it's probably better to define an AssetFilter type and then have a list of filters so that this function doesn't pile up